pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -135,6 +138,23 @@ func (r *AWSLoadBalancerControllerReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, fmt.Errorf("failed to ensure CredentialsRequest for AWSLoadBalancerController %q: %w", req.Name, err)
 	}
 
+	secretProvisioned, err := r.credentialsSecretProvisioned(ctx, credentialsRequest)
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to ensure credentialsrequest secret %q for AWSLoadBalancerController %q: %w", credentialsRequest.Spec.SecretRef.Name, req.Name, err)
```

Comment:
The error message should be different. Probably _failed to verify if credentials secret %q for AWSLoadBalancerController %q has been provisioned_.